viz/pkg/healthcheck/healthcheck.go

```diff
@@ -180,8 +180,25 @@ func (hc *HealthChecker) VizCategory() healthcheck.Category {
 			// "waiting for check to complete" while things converge. If after the timeout
 			// it still hasn't converged, we show the real error (a 503 usually).
 			WithRetryDeadline(hc.RetryDeadline).
-			WithCheckRPC(func(ctx context.Context) (*healthcheckPb.SelfCheckResponse, error) {
-				return hc.vizAPIClient.SelfCheck(ctx, &healthcheckPb.SelfCheckRequest{})
+			WithCheck(func(ctx context.Context) error {
+				results, err := hc.vizAPIClient.SelfCheck(ctx, &pb.SelfCheckRequest{})
```

Comment:
tiny TIOL but I like to leave a little bit of whitespace around code block operations. I find that it helps in code readability:
```diff
diff --git a/viz/pkg/healthcheck/healthcheck.go b/viz/pkg/healthcheck/healthcheck.go
index e6ae2f44..fcf6b7e4 100644
--- a/viz/pkg/healthcheck/healthcheck.go
+++ b/viz/pkg/healthcheck/healthcheck.go
@@ -185,9 +185,11 @@ func (hc *HealthChecker) VizCategory() healthcheck.Category {
                                if err != nil {
                                        return err
                                }
+
                                if len(results.GetResults()) == 0 {
                                        return errors.New("No results returned")
                                }
+
                                errs := []string{}
                                for _, res := range results.GetResults() {
                                        if res.GetStatus() != pb.CheckStatus_OK {
@@ -197,6 +199,7 @@ func (hc *HealthChecker) VizCategory() healthcheck.Category {
                                if len(errs) == 0 {
                                        return nil
                                }
+
                                errsStr := strings.Join(errs, "\n    ")
                                return errors.New(errsStr)
                        }),
```

Reply:
Will do :+1: